ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -434,6 +434,7 @@ object BigDiffy extends Command with Serializable {
         |  --unorderedFieldKey=<key>        ',' separated list of keys for fields which are unordered nested records. Mappings use ':'
         |                                   For example --unorderedFieldKey=fieldPath:fieldKey,otherPath:otherKey
         |  [--with-header]                  Output all TSVs with header rows. Defaults to false
+        |  [--convert-nulls]                Convert nulls to "null" for use in key fields. Defaults to false
```

Comment:
We should match existing behaviour with Avro and just have this enabled in the lib instead of being a CLI arg. We can update internal docs/SO afterwards to reflect this.